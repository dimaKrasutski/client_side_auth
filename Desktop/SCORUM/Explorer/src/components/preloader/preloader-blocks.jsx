import React, { Fragment } from 'react';
import './preloader-blocks.scss';

export const PreloaderRow = () => (
  <tr className="preloader-blocks preloader-row">
    <td>
      <div className="preloader preloader-sm" />
    </td>
    <td>
      <div className="preloader preloader-date" />
    </td>
    <td>
      <div className="preloader" />
    </td>
    <td>
      <div className="preloader preloader-sm float-right" />
    </td>
  </tr>
);

export const PreloaderBlocks = () => (
  <Fragment>
    <PreloaderRow />
    <PreloaderRow />
    <PreloaderRow />
  </Fragment>
);
