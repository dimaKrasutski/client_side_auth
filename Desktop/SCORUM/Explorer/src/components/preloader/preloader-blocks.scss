.preloader-blocks {
  .preloader-sm {
    width: 55px;
  }

  .preloader-date {
    width: 130px;
  }
}
