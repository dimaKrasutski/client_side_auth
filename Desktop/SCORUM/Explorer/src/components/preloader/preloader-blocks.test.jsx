import React from 'react';
import { PreloaderBlocks, PreloaderRow } from './preloader-blocks';

describe('<PreloaderBlocks />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<PreloaderBlocks />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('<PreloaderRow />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<PreloaderRow />);
    expect(wrapper).toMatchSnapshot();
  });
});
