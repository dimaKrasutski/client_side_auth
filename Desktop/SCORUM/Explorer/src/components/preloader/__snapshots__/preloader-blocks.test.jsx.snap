// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreloaderBlocks /> should match snapshot 1`] = `
<React.Fragment>
  <PreloaderRow />
  <PreloaderRow />
  <PreloaderRow />
</React.Fragment>
`;

exports[`<PreloaderRow /> should match snapshot 1`] = `
<tr
  className="preloader-blocks preloader-row"
>
  <td>
    <div
      className="preloader preloader-sm"
    />
  </td>
  <td>
    <div
      className="preloader preloader-date"
    />
  </td>
  <td>
    <div
      className="preloader"
    />
  </td>
  <td>
    <div
      className="preloader preloader-sm float-right"
    />
  </td>
</tr>
`;
